### PR TITLE
Inline parquet-scrooge

### PR DESCRIPTION
**Pull request by ianoc**
Required a little bit of manual code in the Build.sbt since the scrooge sbt plugin doesn't seem to generate multiple languages at once easily.

### CODE REVIEW

File: scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2012 Twitter, Inc.
```
Comment (Gabriel439) [inline scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java]: Shouldn't we update these copyright notices to 2015?

Comment (ianoc) [inline scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java]: Half the scalding repo has 2012. How about I do a follow up PR after this and update all of them?

Comment (Gabriel439) [inline scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java]: Sounds good

Comment (johnynek) [inline scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java]: actually, I don't think we need to keep adding copyright notices. Seems like make work, and the repo LICENSE should be enough (I think @travisbrown signed off on this statement).

Comment (ianoc) [inline scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java]: Oh, ok cool.

Comment (isnotinvain) [inline scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java]: Yeah, but we *do* probably need to leave the license headers as-is from files that we are copying from apache projects

**Comment by isnotinvain** [comment]
LGTM, lets file a ticket for future cleanup though. This will probably just get deleted once we have a more efficient parquet thrift implementation (inside scalding rather than parquet)

**Comment by tsdeng** [comment]
+ 1

**Pull request merged**